index.js

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env node
-const readPkgUp = require("read-pkg-up");
-(async () => {
-  const package = (await readPkgUp()).pkg;
-  Object.entries(package.scripts).forEach(([key, script]) => {
-    console.log(`${key} => ${script}`)
-  })
-})();
+const readPkgUp = require('read-pkg-up');
+const pkgUp = require('pkg-up');
+const packageInfo = readPkgUp.sync();
+
+const package = packageInfo.pkg;
+
+if (
+  typeof package.scripts !== 'object' ||
+  Object.keys(package.scripts).length === 0
+) {
+  console.log(`Package does not contain 'scripts'\n  ${pkgUp.sync()}`);
```

Comment:
packageInfo should have the file path I think
```suggestion
  console.log(`Package does not contain 'scripts'\n  ${packageInfo.path}`);
```